srcpkgs/liburing/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'liburing'
+pkgname=liburing
+version=0.1
+revision=1
+archs="i686* x86_64*"
```

Comment:
What is specific to these archs?

Reply:
The memory barriers (include/liburing/barrier.h) are implemented only for those two archs currently, and as such it will fail compilation under any other arch due to missing definition of smp_mb().

Reply:
Ah, right.